nextjs-app/components/pages/main/WhatWeBring.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Button from "@/components/common/Button";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+
+interface GoalCardProps {
+  imageSrc: string;
+  title: string;
+  subTitle: string;
+  description: string;
+}
+
+const GoalCard = (props: GoalCardProps) => {
+  const { imageSrc, title, subTitle, description } = props;
+  return (
+    <div className="flex flex-col items-center rounded-3 transition-all duration-300 border border-[#7E7059] bg-white z-10">
+      <div className="h-[30px]">
+        <Image
+          src={imageSrc}
+          alt={title}
+          width={120}
+          height={120}
+          className="-translate-y-1/2"
+        />
+      </div>
+      <div className="px-7 pt-11 pb-8 space-y-6">
```

Comment:
```suggestion
      <div className="px-7 pt-11 pb-8 space-y-4">
```

<img width="586" alt="image" src="https://github.com/user-attachments/assets/b757d2c6-8015-45bb-8728-4a412e4c4676" />

Reply:
Oops, my bad! Thanks for pointing that out!